MozillaFoundation/foundation.mozilla.org
# Copy RCC Pages

# Description

Copy RCC page types.

Link to sample test page:
Related PRs/issues: #10424 

# Checklist



**Tests**
- [X] Is the code I'm adding covered by tests? (We still need to add tests for `RCCAuthorIndexPage` but they will be added once the relationships are implemented).

**Changes in Models:**

**Documentation:**

**Testing**

1. Go to `en/responsible-computing-challenge-playbook` and navigate through the pages of the RCC playbook
2. Make sure to tick the AC in #10424
